rotateImage.py

```diff
@@ -0,0 +1,53 @@
+import cv2
+
+# read image as grey scale
+
+def rotateFunction(img):
+
+    # img = cv2.imread(imagePath)
+    # get image height, width
+    (h, w) = img.shape[:2
+             ]
+    # calculate the center of the image
+    center = (w / 2, h / 2)
+
+    angle90 = 90
+    angle180 = 180
+    angle270 = 270
```

Comment:
Hey @kshitijkhune 
I saw that you have rotated the image by 90 degrees and 180 degrees but I am unable to understand the reason behind 270 degrees in this line. Can you please explain?
I think you wanted to remove 270 degrees functionality but you forgot to remove this. If I am wrong can you please clarify and if not then can you please correct it and squash the commits in PR.

Reply:
Thank you for taking out time.Code for 270 is also written but it has been commented out in line 47 48 49. un-commenting it would solve the issue.

Reply:
@kshitijkhune
Will you send the PR again with the changes?